src/components/Headbar.tsx

```diff
@@ -106,7 +106,7 @@ export function Headbar () {
     <div className={'app-headbar h-14 flex justify-center'}>
       <div className={'flex items-center justify-between w-full'}>
         <div className={'flex items-center h-full flex-1'}>
-          <Link to={'/'} className={'app-logo-link mr-2'}></Link>
+          <Link to={'/'} className={'app-logo-link mr-2'} aria-label={`Logseq's logo`}></Link>
```

Comment:
In this case the `aria-label` should describe the link, because there is no readable content in the anchor element. `Home` or `Main` might be a better option. `Logseq's logo` would be a better fit for the `alt` attribute of an image, but that can't be applied here since the image is set as background. Also note that using the word `logo` is usually discouraged. Just using `Logseq` could also be nice middle ground solution, that does not require adding extra markup within the anchor.

Reply:
In fact, I think it would be better to remove the link, otherwise people using screen readers would hear two consecutive identical or different links to the same content.

Reply:
That would also work. The only downside is that users are used to click on the logo to return to home page.

Reply:
> That would also work. The only downside is that users are used to click on the logo to return to home page.

The home link is very close to the logo, so I don't think it's a big problem. Or we can choose to remove the home link and keep the logo link.

Reply:
The button being close is not a big factor here.
Most users will use muscle memory and assume clicking on the logo will take you home, as it does in most sites. 

I think we should keep both.

Reply:
We can merge this as it is. It's good enough for now, and I plan to submit an a11y related PR soon. We can re-evaluate this later.